src/monattack.cpp

```diff
@@ -2770,6 +2773,20 @@ bool mattack::ranged_pull( monster *z )
 
     z->moves -= 150;
 
+    const optional_vpart_position veh_part = here.veh_at( target->pos() );
+    if( foe->in_vehicle && veh_part ) {
+        const cata::optional<vpart_reference> vp_seatbelt = veh_part.avail_part_with_feature( "SEATBELT" );
+        if( vp_seatbelt ) {
+            z->moves -= 200;
+            game_message_type msg_type = foe && foe->is_avatar() ? m_warning : m_info;
+            target->add_msg_player_or_npc( msg_type, _( "%1s tries to drag you, but is stopped by your %2s!" ),
+                                           _( "%1s tries to drag <npcname>, but is stopped by their %2s!" ),
+                                           z->disp_name( false, true ), vp_seatbelt->part().name() );
```

Comment:
This needs to deal some damage instead of just being nullified.  It can go to both the attacker and the target.

Reply:
mmkay. in the next commit, I'll make it do some torso damage (to simulate being crushed/squeezed against the belt/seat/whatever) in addition to the message span stuff mentioned in the earlier comments.